From 5556b73d99aa21ed9e50cffbd4923d06d287c302 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:27 -0400
Subject: [redhat] kabi: Adding symbol param_get_ulong (kernel/params.c)

Message-id: <08fa3a6761db47643f001a4b2e02312ea37ad77d.1410510773.git.poros@redhat.com>
Patchwork-id: 95943
O-Subject: [PATCH RHEL7.1 147/233] kabi: Adding symbol param_get_ulong (kernel/params.c)
Bugzilla: 1071098
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Not used directly, used by module_param internally for type ulong.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/param_get_ulong
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/param_get_ulong
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/param_get_ulong

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/param_get_ulong b/redhat/kabi/kabi-module/kabi_ppc64/param_get_ulong
new file mode 100644
index 0000000..373ee7a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/param_get_ulong
@@ -0,0 +1,2 @@
+#1-
+0x03bd889d param_get_ulong vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/param_get_ulong b/redhat/kabi/kabi-module/kabi_s390x/param_get_ulong
new file mode 100644
index 0000000..373ee7a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/param_get_ulong
@@ -0,0 +1,2 @@
+#1-
+0x03bd889d param_get_ulong vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/param_get_ulong b/redhat/kabi/kabi-module/kabi_x86_64/param_get_ulong
new file mode 100644
index 0000000..373ee7a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/param_get_ulong
@@ -0,0 +1,2 @@
+#1-
+0x03bd889d param_get_ulong vmlinux EXPORT_SYMBOL
-- 
1.7.1